CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
   - prefer_last
   - prefer_match_file_name
   - proper_super_calls
+  - avoid_using_debug_print
```

Comment:
Please increment the version instead of adding it to an old version changelog entry